projects/plugins/backup/src/js/components/Admin.js

```diff
@@ -137,26 +138,29 @@ const Admin = () => {
 		return (
 			<Row>
 				<Col lg={ 6 } md={ 4 }>
-					<h2>{ __( 'Where are my backups stored?', 'jetpack-backup' ) }</h2>
+					<h2>{ __( 'Your cloud backups', 'jetpack-backup' ) }</h2>
 					<p>
 						{ __(
 							'All the backups are safely stored in the cloud and available for you at any time on Jetpack.com, with full details about status and content.',
 							'jetpack-backup'
 						) }
 					</p>
-					{ hasBackupPlan() && ! capabilities.includes( 'backup-realtime' ) && (
-						<a
-							class="jp-cut"
-							href={ getRedirectUrl( 'backup-plugin-realtime-upgrade', { site: domain } ) }
-						>
-							<span>
-								{ __(
-									'Your site is updated with new content several times a day',
-									'jetpack-backup'
-								) }
-							</span>
-							<span>{ __( 'Consider upgrading to real-time protection', 'jetpack-backup' ) }</span>
-						</a>
+					{ hasBackupPlan() && (
+						<>
+							<p>
+								<a
+									href={ getRedirectUrl( 'jetpack-backup', { site: domain } ) }
+									target="_blank"
+									rel="noreferrer"
+								>
+									{ __( 'See all your backups', 'jetpack-backup' ) }
+								</a>
+							</p>
+							<MyPlan
+								purchaseType={ 'backup' }
+								redirectUrl={ getRedirectUrl( 'backup-plugin-my-plan', { site: domain } ) }
```

Comment:
This sent me to `https://wordpress.com/plans/my-plan/<site>?site=<site>`. WordPress doesn't handle backup-only sites well, so we should direct to `https://cloud.jetpack.com/purchases/subscriptions/<site>` instead I think.

Reply:
Done! I changed it in the redirect manager